CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # CHANGELOG
 
 ## Unreleased
+
+## v18.4.0.0
+* Upgrade to [Cumulus v18.4.0](https://github.com/nasa/cumulus/releases/tag/v18.4.0)
 * Add 'Docker in Docker' functionality by giving the container access to the
   host's docker engine. This requires running `make docker-in-docker-permissions`
 
```

Comment:
I wonder if we should make a `18.3.3.1` release with these docker-in-docker changes. I guess PI Planning is still happening, but last I heard ASF wasn't planning on going past `18.3.3` for a while. It would be good if we had a compatible release with these changes in it so we can do local development.

Reply:
I'm definitely fine with holding off on this PR until after a `18.3.3.1` release, I can modify my changes accordingly.  I've already used the changes to deploy to all the ORNL environments so I'm in no hurry.

Reply:
Outcomes of PI Planning is anyone on serverless RDS needs to go v18.4.0 -> v18.5.0 before December 31st 2024 as the serverless v1 is EoL. I'm not sure if they have v18.5.0 out yet

Reply:
It sounds like we need to go to 18.4 as well, so I'm fine with skipping a 18.3.3.1 release and just merging this.